client/public/about.html

```diff
@@ -0,0 +1,261 @@
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-sm bg-light navbar-light">
+      <!-- Brand/logo -->
+      <a class="navbar-brand" href="http://localhost:3000/">
+        <img
+          src="dancing.png"
+          alt="logo"
+          style="width: 30px; height: 30px;"
+        />DanceWithMe
+      </a>
+
+      <!-- Links -->
+      <ul class="navbar-nav">
+        <li class="nav-item">
+          <a class="nav-link" href="http://localhost:3000/">Home</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="">About</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="bg-light">
+      <div class="container py-5">
+        <div class="row h-100 align-items-center py-5">
+          <div class="col-lg-6">
+            <h1 class="display-4">Who Are We?</h1>
+            <p class="lead text-muted mb-0">
+              We are a team from Class #6.
+            </p>
+            <p class="lead text-muted">
+              in
+              <a href="https://hackyourfuture.be/" class="text-muted">
+                <u>Hack Your Future Belgium</u></a
+              >
+            </p>
+          </div>
+          <div class="col-lg-6 d-none d-lg-block">
+            <img src="hyf.jpeg" alt="" class="img-fluid rounded" />
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="bg-white py-5">
+      <div class="container py-5">
+        <div class="row align-items-center mb-5">
+          <div class="col-lg-6 order-2 order-lg-1">
+            <i class="fa fa-bar-chart fa-2x mb-3 text-primary"></i>
+            <h2 class="font-weight-light">Our Goal</h2>
+            <p class="font-italic text-muted mb-4">
+              We want to create an alternative page for people who wish for
+              dancing but who can not find out partners. We want to give people
+              a resource with whom they can dance. We always try to get a dance
+              partner through the internet, but soon we notice THERE IS A LOT OF
+              INFORMATION about dance partners. Moreover, this ubiquitous and
+              complicated information consumes energy and takes a lot of time.
+            </p>
+          </div>
+          <div class="col-lg-5 px-5 mx-auto order-1 order-lg-2">
+            <img
+              src="https://res.cloudinary.com/mhmd/image/upload/v1556834139/img-1_e25nvh.jpg"
+              alt=""
+              class="img-fluid mb-4 mb-lg-0"
+            />
+          </div>
+        </div>
+        <div class="row align-items-center">
+          <div class="col-lg-5 px-5 mx-auto">
+            <img
+              src="https://res.cloudinary.com/mhmd/image/upload/v1556834136/img-2_vdgqgn.jpg"
+              alt=""
+              class="img-fluid mb-4 mb-lg-0"
+            />
+          </div>
+          <div class="col-lg-6">
+            <i class="fa fa-leaf fa-2x mb-3 text-primary"></i>
+            <p class="font-italic text-muted mb-4">
```

Comment:
How does DanceWitnMe differentiate from other applications. We present dance events in your profile page, while our powerful matching engine finds the best partner for you! From now on you don't need to waste your precious time to search for dance events and partners!

Reply:
@u-uysal I recommend to replace the paragraph with this